protocol/app/upgrades/v8.0/upgrade.go

```diff
@@ -60,4 +68,40 @@ func migrateAccountplusAccountState(ctx sdk.Context, k accountpluskeeper.Keeper)
 	ctx.Logger().Info("Successfully migrated accountplus AccountState keys")
 }
 
-// TODO: Scaffolding for upgrade: https://linear.app/dydx/issue/OTE-886/v8-upgrade-handler-scaffold
+const (
+	ID_NUM = 200
+)
+
+// Set market, perpetual, and clob ids to a set number
+// This is done so that the ids are consistent for convenience
+func setMarketListingBaseIds(
+	ctx sdk.Context,
+	pricesKeeper pricestypes.PricesKeeper,
+	perpetualsKeeper perptypes.PerpetualsKeeper,
+	clobKeeper clobtypes.ClobKeeper,
+) {
+	// Set all ids to a set number
+	pricesKeeper.SetNextMarketID(ctx, ID_NUM)
+
+	perpetualsKeeper.SetNextPerpetualID(ctx, ID_NUM)
+
+	clobKeeper.SetNextClobPairID(ctx, ID_NUM)
```

Comment:
non-blocking question: Did we ever think about getting all the next ids, then setting it to be the max of them?

Reply:
Could also just emit an event for what the next ids are in the upgrade.